tests/vspec/test_node_removal/test_files/test.vspec

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) 2023 Contributors to COVESA
+#
+# This program and the accompanying materials are made available under the
+# terms of the Mozilla Public License 2.0 which is available at
+# https://www.mozilla.org/en-US/MPL/2.0/
+#
+# SPDX-License-Identifier: MPL-2.0
+
+A:
+  type: branch
+  description: A is a test node
+A.Float:
+  datatype: float
+  type: actuator
+  unit: mm
+  description: A.Float is a leaf of A of datatype float
+A.Int16:
+  datatype: int16
+  type: sensor
+  unit: rpm
+  description: A.Int16 is a leaf of A of datatype int16
+A.String:
+  datatype: string
+  type: sensor
+  description: A.String is a leaf of A of datatype string
+  deprecation: This is test deprecation, let's say it used to be called Str instead String.
```

Comment:
Inconsistent termination of sentences with a period (full stop).

Reply:
thx done will be changed in the next commit